runtime/master/src/main/java/org/apache/nemo/runtime/master/RuntimeMaster.java

```diff
@@ -274,6 +300,8 @@ public void terminate() {
       Thread.currentThread().interrupt();
     }
 
+    saveMetrics();
```

Comment:
This is also related to the NemoDriver.java comment.

Reply:
i moved this function call to nemo driver